### PR TITLE
Shows error message for failed scim2/Bulk requests

### PR DESCRIPTION
**Pull request by hwupathum**
Solves https://github.com/wso2/product-is/issues/16514

### CODE REVIEW

**Comment by hwupathum** [comment]
The files resulting the EsLint check failure has not been edited by this PR

File: apps/console/src/features/core/api/bulk-operations.ts [hunk 1]
```diff
@@ -53,10 +76,28 @@ export const updateResources = (data: BulkDataPropsInterface): Promise<any> => {
...
     return httpClient(requestConfig)
-        .then((response) => {
+        .then((response: HttpResponse) => {
+            const bulkResponse: BulkResponseInterface | undefined = response?.data;
+            let isSuccessful: boolean = true;
+
+            if (bulkResponse?.schemas?.includes("urn:ietf:params:scim:api:messages:2.0:BulkResponse")) {
+                bulkResponse?.Operations?.some((operation: BulkResponseOperationInterface) => {
+                    // Check if status code exists and if it is not 200.
+                    if (operation?.status?.code !== undefined && operation.status.code !== 200) {
```
Comment (DonOmalVindula) [inline apps/console/src/features/core/api/bulk-operations.ts:86]:
```suggestion
                    if (operation?.status?.code !== 200) {
```

Comment (hwupathum) [reply apps/console/src/features/core/api/bulk-operations.ts:86]: As I tested, those are not equal. I can use 
`!isNaN(operation?.status?.code) && operation.status.code !== 200` though

**Review by brionmario — DISMISSED** [review]

File: apps/console/src/features/core/api/bulk-operations.ts [hunk 2]
```diff
@@ -53,10 +76,30 @@ export const updateResources = (data: BulkDataPropsInterface): Promise<any> => {
...
+        .then((response: HttpResponse) => {
+            const bulkResponse: BulkResponseInterface | undefined = response?.data;
+            let isSuccessful: boolean = true;
+
+            if (bulkResponse?.schemas?.includes("urn:ietf:params:scim:api:messages:2.0:BulkResponse")) {
+                // Show an error if any of the operations returned an error.
+                bulkResponse?.Operations?.some((operation: BulkResponseOperationInterface) => {
+                    const statusCode: number = operation?.status?.code;
+
+                    if (statusCode != 200 && statusCode != 201 && statusCode != 204) {
```
Comment (DonOmalVindula) [inline apps/console/src/features/core/api/bulk-operations.ts:88]:
```suggestion
                    if (statusCode !== 200 && statusCode !== 201 && statusCode !== 204) {
```

**Review by DonOmalVindula — APPROVED** [review]

**Pull request merged**